catalog-rest-service/src/main/resources/ui/src/constants/constants.ts

```diff
@@ -153,6 +158,15 @@ export const getDatasetVersionPath = (datasetFQN: string, version: string) => {
   return path;
 };
 
+export const getDatasetTabPath = (datasetFQN: string, tab = 'schema') => {
```

Comment:
Why do we need separate method for this
We can simply add optional argument to existing method and create the path based on that

Same thing for other added methods as well

Reply:
updated. for topic, dashboard, and pipeline.
- dataset has different things like search column and version so we need a separate method for it.

Reply:
I don't get it, we have separate methods for search column, version etc. but for tab it can be handles in same method right?

Reply:
we only have a separate method for versioning, but the search column thing is handled in the original method that is `getDatasetDetailsPath`. that's why I have added a separate method to handle tab thing

Reply:
Okay